utils/config.ts

```diff
@@ -0,0 +1,7 @@
+declare const CONFIG: {
```

Comment:
How is this generated based on dev, stage and prod?

Reply:
This will be done inside the config folder where we add declarative config files, this is already done in ticket 43 which was assigned to Rajat sir, he hasn't merged his pr for ticket 43 yet, I'll remind him to merge it or I'll set up it soon.